src/transformers/integrations.py

```diff
@@ -1380,7 +1435,6 @@ def setup(self, args, state, model, tokenizer, **kwargs):
             if self._clearml_task is None:
                 self._clearml_task = self._clearml.Task.init(
                     project_name=os.getenv("CLEARML_PROJECT", "HuggingFace Transformers"),
-                    task_name=os.getenv("CLEARML_TASK", "Trainer"),
```

Comment:
Not sure why there is this change added here.

Reply:
That's an accident - it shouldn't have been removed! Reverting right away.